Plugins/Player/Player.cpp

```diff
@@ -1336,4 +1344,171 @@ ArgumentStack Player::SetResManOverride(ArgumentStack&& args)
     return Services::Events::Arguments();
 }
 
+
+CExoLocString CreateCExoLocString(const std::string& str)
+{
+    CExoLocString locStr;
+    locStr.AddString(0,CExoString(str.c_str()),0);
+    return locStr;
+}
```

Comment:
This should go in NWNXLib/Utils.hpp/cpp as the following:

```c
CExoLocString CreateLocString(const std::string& str, int32_t nID = 0, uint8_t bGender = 0);

CExoLocString CreateLocString(const std::string& str, int32_t nID, uint8_t bGender)
{
    CExoLocString locStr;
    locStr.AddString(nID, CExoString(str.c_str()), bGender);
    
    return locStr;
}
```

Reply:
Have moved it to Utils as requested